src/views/components/trace/trace-detail.vue

```diff
@@ -29,22 +29,32 @@
           <option v-for="i in current.traceIds" :value="i" :key="i">{{i}}</option>
         </select>
       </div>
-      <a class="rk-btn sm r" :class="{'ghost':mode}" @click="mode = false">
+
```

Comment:
![image](https://user-images.githubusercontent.com/19775780/62411477-e85d8900-b625-11e9-975c-a60149216115.png)
The color is incurrect, active is blue and unactive is dark